client/src/pages/Login.js

```diff
@@ -64,45 +63,44 @@ const SocialButton = styled(Button).attrs((props) => ({
 `;
 
 export default ({ isLoginForm }) => {
-  
   const dispatch = useDispatch();
 
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
 
-  const handleInputChangeEmail = e => {
+  const handleInputChangeEmail = (e) => {
     setEmail(e.target.value);
-  }
+  };
 
-  const handleInputChangePassword = e => {
+  const handleInputChangePassword = (e) => {
     setPassword(e.target.value);
-  }
+  };
 
   const handleLoginWithEmail = (evt) => {
     evt.preventDefault();
-    if (!validateEmail (email)) {
-      Toast.fail('Invalid email address!', 3);
+    if (!validateEmail(email)) {
+      Toast.fail("Invalid email address!", 3);
       return;
     }
     if (password.length < PASSWORD_MIN_LENGTH) {
```

Comment:
What about creating a password validation function? Just like you did with email! Maybe return the error message too

Reply:
yeah `validatePassword` would be a good idea, just have to make sure it has the same validation rules as backend / Auth0.
I consider this out of scope of this PR though.

Reply:
About that, it seems like there was first a `PasswordInput` too but it wasn't working properly so currently it's unused. We should clean that up as well.